mllm/engine/HpcThreadPool.hpp

```
@@ -13,7 +13,10 @@
 #include <functional>
 #include <condition_variable>
 
-#define MLLM_HPC_THREAD_POOL_TASK_LIMITS 2
+// One task slot per expected concurrent op. 8 covers the GDN recurrence
+// 8-lane cap plus sibling parallel ops; too small a limit makes splitTask fall
+// back to serial on the main thread under multi-layer concurrency.
+#define MLLM_HPC_THREAD_POOL_TASK_LIMITS 8
```

Clang (14.0.6)

[error] replace macro with enum

(modernize-macro-to-enum,-warnings-as-errors)